dockerfiles/framework/scale/bootstrap.py

```diff
@@ -286,6 +299,11 @@ def deploy_webserver(client, app_name, es_url, db_url, broker_url):
         'DCOS_PACKAGE_FRAMEWORK_NAME': FRAMEWORK_NAME,
         'DCOS_SERVICE_ACCOUNT': str(secrets_dcos_sa),
         'ENABLE_WEBSERVER': 'true',
+        'FLUENTD_BUFFER_WARN': str(FLUENTD_BUFFER_WARN),
```

Comment:
Any of these new environment variables being added need to be documented in the table at the bottom of the `scale/README.md` file. I certainly have no idea what the difference is between BUFFER_WARN and BUFFER_SIZE_WARN... We need some explanation of what they mean.